package.json

```diff
@@ -27,8 +27,10 @@
     "iconv-lite": "0.4.13",
     "minimist": "^1.2.0",
     "native-keymap": "0.1.2",
+    "pty.js": "git+https://github.com/jeremyramin/pty.js.git#28f2667",
     "sax": "1.1.2",
```

Comment:
@bpasero You raised concerns about NAN versions but those packages are prebuilt so it shouldn't be a problem, right?